### PR TITLE
Restore SDK scripts

### PR DESCRIPTION
**Pull request by wolf31o2**
These are pulled from `release/3.6` and are fallbacks for the current scripts while we shake out any bugs in the `cdap` script.

### CODE REVIEW

File: cdap-standalone/pom.xml [hunk 1]
```diff
@@ -535,6 +535,7 @@
                         <include>cdap</include>
                         <include>functions.sh</include>
                         <include>cdap-cli.bat</include>
+                        <include>cdap-cli.sh</include>
```
Comment (sreevatsanraman) [inline cdap-standalone/pom.xml:538]: We need to include cdap.sh too.

Comment (wolf31o2) [reply cdap-standalone/pom.xml:538]: It's already pulled in from above. Lines 329-332

Comment (sreevatsanraman) [reply cdap-standalone/pom.xml:538]: 👍

**Comment by sreevatsanraman** [comment]
LGTM

**Pull request merged**